extensions/amp-jwplayer/0.1/amp-jwplayer.js

```diff
@@ -287,6 +287,11 @@ class AmpJWPlayer extends AMP.BaseElement {
     this.contentBackfill_ = element.getAttribute('data-content-backfill') || '';
     this.contentRecency_ = element.getAttribute('data-content-recency') || '';
 
+    // Allow outstream playback
+    if (this.contentid_ === 'outstream') {
```

Comment:
Can we avoid adding a generic remapping of 'outstream' to 'oi7pAMI1'?

This appears to be a value that could be specified as the consent id.

Reply:
`oi7pAMI1` is our content-id for all outstream players. This substitution is simply so customers don't have to memorize an arbitrary string.